pallets/connectors/src/message.rs

```diff
@@ -201,6 +201,39 @@ where
 		tranche_id: TrancheId,
 		investor: Address,
 	},
+	/// The message sent back to the domain from which a `CollectInvest` message
+	/// has been received, which will ensure the `investor` gets the payout
+	/// respective to their investment.
+	///
+	/// Directionality: Centrifuge -> EVM Domain.
+	ExecutedCollectInvest {
+		pool_id: PoolId,
+		tranche_id: TrancheId,
+		investor: Address,
+		/// The currency in which the payout takes place
+		currency: u128,
+		/// The amount being paid out to the investor
+		payout: Balance,
+		/// The remaining amount the investor can still collect
+		remaining: Balance,
+	},
+
+	/// The message sent back to the domain from which a `CollectRedeem` message
+	/// has been received, which will ensure the `investor` gets the payout
+	/// respective to their redemption.
+	///
+	/// Directionality: Centrifuge -> EVM Domain.
+	ExecutedCollectRedeem {
+		pool_id: PoolId,
+		tranche_id: TrancheId,
+		investor: Address,
+		/// The currency in which the payout takes place
+		currency: u128,
+		/// The amount being paid out to the investor
+		payout: Balance,
+		/// The remaining amount the investor can still collect
+		remaining: Balance,
+	},
```

Comment:
The docs are not correct and if we implement it like said in the docs it is not the expected information that this message should signal. Especially, for `ERC-4626`. 

Please [see ](https://github.com/centrifuge/centrifuge-chain/issues/1356)

Reply:
This is why I would be really specific with the names.

Reply:
> For sure. It is not the remaining amount one can still collect, but rather the amount one has still "locked", either stable coin for investing or tranche token for redeeming.

Right, so that was my understanding as well. When I wrote "... the investor can still collect" I had in mind that it was an amount that would be executed at a later epoch. But if you are absolutely right that it's not the right choice of words.

> This is why I would be really specific with the names

I want to find common ground here, so let's try to work this out. For _me_,  e.g. `remaining_investment_invest` is not any clearer, it's just repeating words that are already in context (`Invest` in `ExecutedCollectInvest`, same for `Redeem` in `ExecutedCollectRedeem`).

Would it help to name it `remaining_locked_investment` and `remaining_locked_redeem`? Don't mean to introduce new language into the table but usually the language used when explaining this stuff low key to each other is usually helpful. Let me know your thoughts 👍

Reply:
And thanks for the careful review dissecting the `///` docs, it's really important to get all this stuff right both from a code and from an understanding perspective!

Reply:
Naming changes sound good!

Reply:
I think `currency` is also redundant for `ExecutedCollectInvest`, since the currency that's being paid out is the tranche token, which is already implicit through the `pool_id` and `tranche_id` parameters.

Reply:
> I think `currency` is also redundant for `ExecutedCollectInvest`, since the currency that's being paid out is the tranche token, which is already implicit through the `pool_id` and `tranche_id` parameters.

@mustermeiszer can you comment on this? 👀